src/registrar/views/domain.py

```diff
@@ -822,6 +822,18 @@ def form_valid(self, form):
 class DomainInvitationDeleteView(SuccessMessageMixin, DomainInvitationPermissionDeleteView):
     object: DomainInvitation  # workaround for type mismatch in DeleteView
 
+    def post(self, request, *args, **kwargs):
```

Comment:
(Optional): This code is unit testable. It's optional because it should be extremely hard if not impossible for a user to stumble upon this condition.